assets/js/donut.js

```diff
@@ -61,20 +59,14 @@ label.append("tspan")
 	.attr("dy", "1.1em")
 	.text("complete");
 
-// building a legend
-svg.append('g')
-    .attr('class', 'legend')
-    .selectAll('text')
-    .data(slices)
-    .enter()
-    .append('text')
-    .text(function (d) {
-        return d.data.value + ' hours - ' + d.data.label;
-    })
-    .attr('fill', function (d) {
-        return color(d.data.label);
-    })
-    .attr('y', function (d, i) {
-        return 20 * (i + 1);
-    })
-    .attr('x', 300);
+(function ($) {
```

Comment:
- assuming using jquery at this point is because default legend functionality in d3js makes it part of the svg image preventing separation of legend and 'donut'?

Reply:
Yes. My thinking was this makes it easier to control long term and potentially more accessible.